Tests/MockoloTestCase.swift

```diff
@@ -54,18 +54,35 @@ class MockoloTestCase: XCTestCase {
         }
     }
 
-    func verify(srcContent: String, mockContent: String? = nil, dstContent: String, header: String = "", declType: DeclType = .protocolType, useTemplateFunc: Bool = false, useMockObservable: Bool = false, testableImports: [String] = [], allowSetCallCount: Bool = false, mockFinal: Bool = false, enableFuncArgsHistory: Bool = false, concurrencyLimit: Int? = 1, disableCombineDefaultValues: Bool = false) {
+    func verify(srcContent: String, mockContent: String? = nil, dstContent: String, header: String = "", declType: DeclType = .protocolType, useTemplateFunc: Bool = false, useMockObservable: Bool = false, testableImports: [String] = [], allowSetCallCount: Bool = false, mockFinal: Bool = false, enableFuncArgsHistory: Bool = false, outputFilePath: String? = nil, concurrencyLimit: Int? = 1, disableCombineDefaultValues: Bool = false) {
+        let outputFilePath = outputFilePath ?? dstFilePath
```

Comment:
[nits]
The following naming seems better:
`outputFilePath` → `dstFilePath`
`dstFilePath` → `defaultDstFilePath`

Reply:
@uhooi Thanks for your comment! I totally agree with that. Fixed by https://github.com/uber/mockolo/pull/193/commits/ee1f00fc7ac31105e8e8c910c7895e7b000545c4 .